internal/commands/test.go

```diff
@@ -485,13 +486,18 @@ func parseFileList(fileList []string) ([]string, error) {
 	return files, nil
 }
 
-func getFilesFromDirectory(directory string) ([]string, error) {
+func getFilesFromDirectory(directory string, exceptions string) ([]string, error) {
 	var files []string
-	err := filepath.Walk(directory, func(currentPath string, info os.FileInfo, err error) error {
+	regexp := regexp.MustCompile(exceptions)
```

Comment:
nit: I feel we should use .Compile() here and handle the error in a more graceful way. If the user passes in a bad regex, this is going to panic.

Reply:
agree on that, I've put an error handling to eliminate panic